### DISCUSSION TITLE
how to integrate Shopvox with Swell Commerce

### DISCUSSION THREAD

**Discussion by RushiNCode93** [swellstores/community#139] [Help] [answered]
how to integrate Shopvox with Swell Commerce? is this possible to do? if yes then please guide me on this.

**Comment by sags95** [comment, marked as answer]
@RushiNCode93  As there is no pre-built integration for Shopvox, you would have to use the [shopVOX API](https://shopvoxapi.docs.apiary.io/#reference) in combination with [webhooks ](https://www.swell.is/help/store-setup/developer-tools#webhooks). 

So new orders created in Swell can be created in Shopvox as sales orders. Depending on the scope of the integration this could be adapted or modified.

Otherwise if you wanted to sync data from Shopvox into Swell, you would utilize [webhooks from Shopvox](https://shopvoxapi.docs.apiary.io/#reference/webhooks/create) and the [Backend API ](https://developers.swell.is/backend-api/introduction)in Swell. So new orders or quotes created in Shopvox can be synced with Swell as Orders or other models.

Swell's architecture is extremely flexible, so if you wanted to create other data models in Swell (such as work_orders, sales_orders, quotes) then you can [create custom models](https://www.swell.is/help/store-setup/model-editor-and-custom-fields#model-editor-and-custom-fields)
